views/table.ejs

```diff
@@ -0,0 +1,151 @@
+<head>
+	<title>Events</title>
+	<!-- Bootstrap CSS -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+</head>
+<br>
+<a href="/event/view/<%=event_id%>"><button class="btn btn-primary">Return to event</button> </a>
+<body class="container">		
+	<br>
+	<div style="text-align: center;font-size:2em;"><%=eventName%> - Table Plan</div>
+	<br>
+	<% if (succ && succ.length > 0) { %>
```

Comment:
For consistency would be best to have flash messages at the top as they show below page header